sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_visits_daily_v2/query.sql

```diff
@@ -0,0 +1,284 @@
+  )
+  AND LOGICAL_OR(is_default_ui) AS is_sponsored_content_interaction,
+  LOGICAL_OR(
+    event_category = 'pocket'
+    AND event_name IN ('click')
+    AND SAFE_CAST(mozfun.map.get_key(event_details, 'is_sponsored') AS BOOLEAN)
+  )
+  AND LOGICAL_OR(is_default_ui) AS is_sponsored_content_click,
+  LOGICAL_OR(
+    event_category = 'pocket'
+    AND event_name IN ('impression')
+    AND SAFE_CAST(mozfun.map.get_key(event_details, 'is_sponsored') AS BOOLEAN)
+  )
+  AND LOGICAL_OR(is_default_ui) AS is_sponsored_content_impression,
+  IF(
+    LOGICAL_OR(is_default_ui),
+    COUNTIF(event_category = 'pocket' AND event_name IN ('click')),
+    0
+  ) AS any_content_click_count,
+  IF(
+    LOGICAL_OR(is_default_ui),
+    COUNTIF(event_category = 'pocket' AND event_name IN ('impression')),
+    0
+  ) AS any_content_impression_count,
+  IF(
+    LOGICAL_OR(is_default_ui),
+    COUNTIF(
+      event_category = 'pocket'
+      AND event_name IN ('click')
+      AND NOT SAFE_CAST(mozfun.map.get_key(event_details, 'is_sponsored') AS BOOLEAN)
+    ),
+    0
+  ) AS organic_content_click_count,
+  IF(
+    LOGICAL_OR(is_default_ui),
+    COUNTIF(
+      event_category = 'pocket'
+      AND event_name IN ('impression')
+      AND NOT SAFE_CAST(mozfun.map.get_key(event_details, 'is_sponsored') AS BOOLEAN)
+    ),
+    0
+  ) AS organic_content_impression_count,
+  IF(
+    LOGICAL_OR(is_default_ui),
+    COUNTIF(
+      event_category = 'pocket'
+      AND event_name IN ('click')
+      AND SAFE_CAST(mozfun.map.get_key(event_details, 'is_sponsored') AS BOOLEAN)
+    ),
+    0
+  ) AS sponsored_content_click_count,
+  IF(
+    LOGICAL_OR(is_default_ui),
+    COUNTIF(
+      event_category = 'pocket'
+      AND event_name IN ('impression')
+      AND SAFE_CAST(mozfun.map.get_key(event_details, 'is_sponsored') AS BOOLEAN)
+    ),
+    0
+  ) AS sponsored_content_impression_count,
+  LOGICAL_OR(event_category = 'topsites' AND event_name IN ('click', 'dismiss'))
+  AND LOGICAL_OR(is_default_ui) AS is_topsites_interaction,
+  LOGICAL_OR(event_category = 'topsites' AND event_name IN ('click'))
+  AND LOGICAL_OR(is_default_ui) AS is_topsites_click,
+  LOGICAL_OR(event_category = 'topsites' AND event_name IN ('impression'))
+  AND LOGICAL_OR(is_default_ui) AS is_topsites_impression,
+  LOGICAL_OR(
+    event_category = 'topsites'
+    AND event_name IN ('click', 'dismiss')
+    AND SAFE_CAST(mozfun.map.get_key(event_details, 'is_sponsored') AS BOOLEAN)
+  )
+  AND LOGICAL_OR(is_default_ui) AS is_sponsored_topsites_interaction,
+  LOGICAL_OR(
+    event_category = 'topsites'
+    AND event_name IN ('click')
+    AND SAFE_CAST(mozfun.map.get_key(event_details, 'is_sponsored') AS BOOLEAN)
+  )
+  AND LOGICAL_OR(is_default_ui) AS is_sponsored_topsites_click,
+  LOGICAL_OR(
+    event_category = 'topsites'
+    AND event_name IN ('impression')
+    AND SAFE_CAST(mozfun.map.get_key(event_details, 'is_sponsored') AS BOOLEAN)
+  )
```

Comment:
Consolidate into a single `LOGICAL_OR`, see comment for line 95

Reply:
We cannot do that for reasons explained above